packages/erxes-ui/src/utils/core.tsx

```
@@ -196,10 +314,10 @@ export const reorder = (
 export const generateRandomColorCode = () => {
   return `#${Math.random().toString(16).slice(2, 8)}`;
 };
-const isNumeric = n => {
+const isNumeric = (n) => {
   return !isNaN(parseFloat(n)) && isFinite(n);
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)